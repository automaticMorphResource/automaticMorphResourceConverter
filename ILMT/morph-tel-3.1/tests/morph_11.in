1	waracU	unk	
1	gummadini	unk	
1	wIsukovadaM	unk	
1	valana	unk	
1	gyAstrik	unk	
1	nuMci	unk	
1	upaSamanaM	unk	
1	laBiswuMxi	unk	
1	.	unk	
