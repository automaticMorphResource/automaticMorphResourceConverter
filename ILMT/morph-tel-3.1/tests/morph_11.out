1	waracU	unk	<fs af='waracU,avy,,,,,,'>
2	gummadini	unk	<fs af='gummadi,n,,sg,,,nu,nu'>|<fs af='gummadi,n,,sg,,,ni,ni'>
3	wIsukovadaM	unk	<fs af='wIsuko,v,any,any,any,,adaM,adaM'>|<fs af='wIsuko,v,any,any,any,,adaM,adaM'>
4	valana	unk	<fs af='valana,avy,,,,,0,0_avy'>|<fs af='vala,n,,sg,,,na,na'>
5	gyAstrik	unk	<fs af='gyAstrik,n,,sg,,d,0,0'>|<fs af='gyAstrik,n,,sg,,o,ti,ti'>
6	nuMci	unk	<fs af='nuMci,avy,,,,,0,0_avy'>
7	upaSamanaM	unk	<fs af='upaSamanaM,n,,sg,,d,0,0'>
8	laBiswuMxi	unk	<fs af='laBiMcu,v,fn,sg,3,,wA,wA'>
9	.	unk	<fs af='&dot;,punc,,,,,,'>
